.github/workflows/vulnerabilities.yml

```diff
@@ -1,17 +1,25 @@
-name: check vulnerable nuget packages
-
-on: [push, pull_request]
+name: List vulnerable packages
 
+on:
+  schedule:
+    # once a day
+    - cron:  '0 0 * * *'
 jobs:
-  check-vulns:
-    runs-on: ubuntu-latest
-
+  List-vulnerable-packages:
+    runs-on: windows-latest
+    env:
+      DOTNET_CLI_TELEMETRY_OPTOUT: 1
+      DOTNET_NOLOGO: 1
+      DOTNET_MULTILEVEL_LOOKUP: 1
```

Comment:
This is the default, according to [this](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-environment-variables#dotnet_multilevel_lookup).  Though some changes coming for .NET 7.  Is there a reason it needs to be specified here explicitly?